src/main/java/org/junit/rules/MethodRule.java

```diff
@@ -14,14 +14,7 @@
  * or write your own:
```

Comment:
Instead of using an unnumbered list with a single item, IMHO we should re-formulate the last sentence of this paragraph and include a link to `TestWatchman` there. What do you think?